src/blueberrypy/command.py

```diff
@@ -1,21 +1,26 @@
 import logging
+from importlib import import_module
+
 import os
 import sys
 import re
 import textwrap
 
 from datetime import datetime
 from functools import partial
+from code import InteractiveConsole
 
 import cherrypy
+from IPython.terminal.interactiveshell import TerminalInteractiveShell
```

Comment:
because it's not a runtime dependency, it should be optional; if one wants to use ipython they'll install it.